sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/ShowTablesSchema.scala

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.catalyst.plans
+
+import org.apache.spark.sql.catalyst.expressions.AttributeReference
+import org.apache.spark.sql.types.{BooleanType, StringType, StructType}
+
+private[sql] object ShowTablesSchema {
+  val schema = StructType.fromAttributes(attributes())
```

Comment:
I don't think this is used any longer, and it would use the wrong attributes if it were used. Can you remove it?

Reply:
Removed. Question: `fromAttributes` just takes `name`, `dataType`,`nullable`, and `metaData` from attributes, so how can this use wrong attributes?

Reply:
I think you're right that it would create an identical struct type each time, so it isn't a problem that it uses a different set of attributes. But I think it is still a good practice not to do this, assuming that will always be the case.